UI/obs-app.cpp

```diff
@@ -1446,6 +1446,8 @@ bool OBSApp::OBSInit()
 
 	blog(LOG_INFO, "Browser Hardware Acceleration: %s",
 	     browserHWAccel ? "true" : "false");
+#else
+	blog(LOG_INFO, "Qt Version: %s", qVersion());
```

Comment:
Per my earlier comment on the PR, I'd prefer if this was logged for all platforms.